lib/sport_ngin_aws_auditor/instance.rb

```diff
@@ -24,11 +25,25 @@ def initialize(type, data_array)
         elsif data_array[0] > 0
           self.category = "reserved"
         end
+
+        if region_based?
+          my_match = type.match(/(\w*\s*\w*\s{1})\s*(\s*\S*)/)
```

Comment:
Consider a code comment here with an example value of `type` that this match is acting on for easier readability.